wls-broadcast-service/src/test/java/de/muenchen/oss/wahllokalsystem/broadcastservice/integration/BroadcastIntegrationTest.java

```diff
@@ -0,0 +1,296 @@
+                .isInstanceOf(ServletException.class)
+                .hasMessageContaining("Das Object BroadcastMessage ist nicht vollständig.");
+    }
+
+    @Test
+    public void getMessageIntegrationTest() throws Exception {
+        log.debug("#GetMessageIntegrationTest");
+        messageRepository.save(Testdaten.createMessage("123", "Das ist ein Test", LocalDateTime.now()));
+        // @formatter:off
+        MockHttpServletResponse result =
+                mvc.perform(
+                        get(GETMESSAGE_URL + "123")
+                        .contentType(MediaType.APPLICATION_JSON_UTF8)
+                        .accept(MediaType.APPLICATION_JSON))
+                        .andReturn().getResponse();
+        // @formatter:on
+        String content = result.getContentAsString();
+        Message message = objectMapper.readValue(content, Message.class);
+        Assertions.assertThat(message.getNachricht()).isEqualTo("Das ist ein Test");
+    }
+
+    @Test
+    public void getMessageIntegrationTestGetParamBlank() {
+        log.debug("#GetMessageIntegrationTestGetParamBlan");
+        Exception catchedException1 = null;
+        try {
+            mvc.perform(
+                    get(GETMESSAGE_URL + "    ")
+                            .contentType(MediaType.APPLICATION_JSON_UTF8)
+                            .accept(MediaType.APPLICATION_JSON))
+                    .andReturn().getResponse();
+
+        } catch (Exception e) {
+            catchedException1 = e;
+        }
+
+        Assertions.assertThat(catchedException1)
+                .isNotNull()
+                .isInstanceOf(ServletException.class)
+                .hasMessageContaining("wahlbezirkID is blank or empty");
+    }
+
+    @Test
+    public void getMessageIntegrationTestGetParamEmpty() throws Exception {
+        log.debug("#GetMessageIntegrationTestGetParamEmpty");
+        String wahlbezirkID = "";
+        mvc.perform(get(GETMESSAGE_URL + wahlbezirkID)
+                .contentType(MediaType.APPLICATION_JSON))
+                .andExpect(status().isNotFound())
+                .andExpect(result -> {
+                    Map<String, Object> mmodel = result.getModelAndView().getModel();
+                    Assertions
+                            .assertThat(mmodel)
+                            .containsKey("wlsException")
+                            .extracting("wlsException")
+                            .isInstanceOf(TechnischeWlsException.class)
+                            .extracting("code", "serviceName")
+                            .contains("999", serviceOid);
+
+                    Assertions
+                            .assertThat(mmodel)
+                            .containsKey("wlsException")
+                            .extracting("wlsException.cause")
+                            .isInstanceOf(NoResourceFoundException.class);
+                });
+    }
+
+    @Test
+    public void getMessageNoContentIntegrationTest() {
```

Comment:
Schreibt man mit Junit 5 nicht mehr

```suggestion
    void getMessageNoContentIntegrationTest() {
```